site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -37,7 +37,7 @@ spec:
         game: my-game
       matchExpressions:
         - {key: tier, operator: In, values: [cache]}
-      # [Stage:Alpha]
+      # [Stage:Beta]
```

Comment:
These changes will also need to be `feature` shortcoded, such that they only get updated when we do our next release.

Reply:
I added `feature` shortcode wrapping `{{< tabpane >}}{{< /tabpane >}} `, 
because the document did not build as intended if I added it inside tabpane.